README.md

```diff
@@ -666,4 +667,11 @@ getNodesName.getResponseField('0.name')
 
 ```
 
-In the example presented the response of the call to `/api/nodes` endpoint is an JSON array of objects. For the `RabbitMqBrokerInstance` there will be a single object, whereas for the `RabbitMqBrokerCluster` there will be three objects presenting information for each node. Arrays are flattened by using the index for a position of the object and that is why the name of the first (and in the example only) node will is retrieved by specifying the response field name `0.name`.  
\ No newline at end of file
+In the example presented the response of the call to `/api/nodes` endpoint is an JSON array of objects. For the `RabbitMqBrokerInstance` there will be a single object, whereas for the `RabbitMqBrokerCluster` there will be three objects presenting information for each node. Arrays are flattened by using the index for a position of the object and that is why the name of the first (and in the example only) node will is retrieved by specifying the response field name `0.name`.  
+
+### Additional RabbitMQ Broker Examples
+
+This section includes additional examples and use cases for working with RabbitMQ Brokers using the AWS::AmazonMQ L2+ Construct Library.
```

Comment:
Change _AWS::AmazonMQ L2+ Construct Library_ to `@cdklabs/cdk-amazonmq` or _this library_.

Reply:
Addressed by new changes.